src/neo-vm/FaultState.cs

```diff
@@ -6,6 +8,7 @@ public sealed class FaultState
 
         public bool Rethrow;
         public bool HasCatchableInterrupt;
+        public Stack<CatcheableException> CatchedExceptionStack { get; } = new Stack<CatcheableException>();
```

Comment:
Do we need a stack? Maybe move the exception to the TryStack is enough.

Reply:
😂  Done